modules/react/layout/stories/examples/Flex/FlexCardResProps.tsx

```diff
@@ -0,0 +1,113 @@
+import * as React from 'react';
+import {Flex, Box} from '@workday/canvas-kit-react/layout';
+import {PrimaryButton, SecondaryButton} from '@workday/canvas-kit-react/button';
+import {type} from '@workday/canvas-kit-react/tokens';
+import {
+  useResizeObserver,
+  ResponsiveContextProvider,
+  useResponsiveContainerStyles,
+} from '@workday/canvas-kit-react/common';
+
+// temporary placeholder until type components are added to canvas-kit
+const H3 = props => (
+  <h3
+    style={{
+      ...type.levels.body.large,
+      margin: 0,
+      fontWeight: type.properties.fontWeights.bold,
+    }}
+    {...props}
+  />
+);
```

Comment:
You can use `Text` component here as well.

Reply:
This example will be deleted. It's for testing purposes and was cleaned up with Alan's style props update.